universe/ValueRef.cpp

```diff
@@ -1696,7 +1696,6 @@ int ComplexVariable<int>::Eval(const ScriptingContext& context) const
     if (variable_name == "BuildingTypesOwned" ||
         variable_name == "BuildingTypesProduced" ||
         variable_name == "BuildingTypesScrapped" ||
-        variable_name == "SpeciesShipsDestroyed" ||
         variable_name == "SpeciesColoniesOwned" ||
         variable_name == "SpeciesPlanetsBombed" ||
         variable_name == "SpeciesPlanetsDepoped" ||
```

Comment:
@geoffthemedio Looks like `SpeciesPlanetsOwned` added in 109de48d is not used at all, can i remove it, too?

Reply:
yes

Reply:
Done